cmd/agent-js-worker/src/agent.js

```diff
@@ -0,0 +1,1089 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+'use strict'
+
+const inNode = typeof process !== 'undefined' && process.versions != null && process.versions.node != null
+const inBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined';
+
+// wait for 'notifierWait' milliseconds before retrying to check for incoming notifications
+const notifierWait = 10000
+
+// time out for command operations
+const commandTimeout = 20000
+
+// base path to load assets from at runtime
+const __publicPath = _ => {
+    if (inNode) {
+        // TODO determine module_path at runtime
+        return process.cwd() + "/node_modules/@trustbloc/agent-js-worker/"
+    } else if (inBrowser) {
+        return "/agent-js-worker/"
+    } else {
+        // TODO #1127 - throw error or use default?
```

Comment:
1217 is from aries project,
better a add comment at the top saying this file is copied from aries, users for this js worker should know.

Reply:
ok, that stuff definitely needs additional cleanups - will do that as part of the next task about merging `agent-js-worker` and `trustbloc-agent-js-worker`

Reply:
then it shouldn't be called 'aries-js-worker' once merged

Reply:
it will not, I plan to merge functionality of `trustbloc-agent-js-worker` into `agent-js-worker` so we will end up with the following structure:
```
|-agent-js-worker
|-http-server
|-user-agent 
```